{{cookiecutter.project_name}}/.yamllint

```diff
@@ -0,0 +1 @@
+extends: default
```

Comment:
Is there any way to get rid of this file? Like default configuration or `setup.cfg` or something?

Reply:
I think we can remove it.

Reply:
From docs:

If -c is not provided, yamllint will look for a configuration file in the following locations (by order of preference):

.yamllint in the current working directory
$XDG_CONFIG_HOME/yamllint/config
~/.config/yamllint/config



So remove it from PR?